4-esm-webpack/webpack.config.js

```diff
@@ -1,26 +1,15 @@
-'use strict'
+import { join } from 'path'
 
-const path = require('path')
-
-module.exports = {
+export default {
   target: 'web',
   mode: process.env.NODE_ENV || 'development',
   devtool: 'source-map',
   entry: [
-    './browser.mjs'
+    './browser.js'
   ],
```

Comment:
👆 No extension juggling. With `esm` you can use `.js` for everything _(and it's preferred as you'll get more ecosystem interop enabled)_.